docs/src/cloudformation.rst

```diff
@@ -14,7 +14,7 @@
 AWS CloudFormation Library
 ##########################
 
-The :doc:`AWS Construct Library <aws-construct-lib>` includes constructs with rich APIs
```

Comment:
Why did you remove the link?

Reply:
The link was originally in a different topic/RST file. We renamed aws-construct-lib.rst as cloudformation.rst and moved the text (including the link) into cloudformation.rst. Fortunately, if Sphinx cannot find a link, it just leaves the text. Not sure if we get any Sphinx error/warning during the build.

Reply:
Nope, aws-construct-lib is a totally different topic:

- https://awslabs.github.io/aws-cdk/cloudformation.html
- https://awslabs.github.io/aws-cdk/aws-construct-lib.html

Reply:
I'll add it back.